src/cargo/core/workspace.rs

```diff
@@ -1645,7 +1646,7 @@ pub struct InheritableFields {
     keywords: Option<Vec<String>>,
     categories: Option<Vec<String>>,
     license: Option<String>,
-    license_file: Option<String>,
+    license_file: Option<PathBuf>,
```

Comment:
Is there a reason this was changed or was it left over from the experiment?

Reply:
It was from the experiment. It was done instead of resolving to a path, converting to a string, and then later converting back to a path. If you like the experiment we can keep it. It's up to you.

Reply:
Let's go back